homeassistant/components/cert_expiry/__init__.py

```diff
@@ -46,9 +46,7 @@ async def async_finish_startup(_):
         await async_finish_startup(None)
     else:
         entry.async_on_unload(
-            hass.bus.async_listen_once(
-                EVENT_HOMEASSISTANT_STARTED, async_finish_startup
-            )
+            hass.bus.async_listen(EVENT_HOMEASSISTANT_STARTED, async_finish_startup)
```

Comment:
Side note: I think we've solved this in `async_at_started` which should probably be used here.

Reply:
I think that would be a better solution long term